code/modules/paperwork/paper_biscuit.dm

```diff
@@ -1,28 +1,41 @@
 /obj/item/folder/biscuit
-	name = "\proper biscuit card"
-	desc = "An biscuit card. Has label which says <b>DO NOT DIGEST</b>."
+	name = "biscuit card"
+	desc = "A biscuit card. On the back, <b>DO NOT DIGEST</b> is printed in large lettering."
 	icon_state = "paperbiscuit"
 	bg_color = "#ffffff"
 	w_class = WEIGHT_CLASS_TINY
 	max_integrity = 130
 	drop_sound = 'sound/items/handling/disk_drop.ogg'
 	pickup_sound = 'sound/items/handling/disk_pickup.ogg'
+	contents_hidden = TRUE
 	/// Is biscuit cracked open or not?
 	var/cracked = FALSE
+	/// The paper slip inside, if there is one
+	var/obj/item/paper/paperslip/contained_slip
+
+/obj/item/folder/biscuit/Initialize(mapload)
+	. = ..()
+	if(contained_slip)
+		new contained_slip(src)
```

Comment:
ah here's your issue. we're hanging refs because you don't tie this to the variable on the item, so QDEL_NULL does absolutely nothing because we don't tie it back to this variable. You can fix this by doing

```suggestion
	if(contained_slip)
		contained_slip = new contained_slip(src)
```